spacy_legacy/tests/test_legacy.py

```diff
@@ -15,6 +15,7 @@
     ("architectures", "Tok2Vec.v1"),
     ("layers", "StaticVectors.v1"),
     ("loggers", "WandbLogger.v1"),
+    ("loggers", "ConsoleLogger.v1"),
```

Comment:
```suggestion
    ("loggers", "ConsoleLogger.v1"),
    ("loggers", "WandbLogger.v1"),
```
just to keep this list alphabetically